### PR TITLE
Main page 24h change indication

### PR DESCRIPTION
**Pull request by Ran73Cohen**
#1050

### CODE REVIEW

File: src/store/modules/swap/ethBancor.ts [hunk 1]
```diff
@@ -1469,6 +1469,11 @@ export class EthBancorModule
   get stats() {
     const apiData = this.apiData!;
     const bntUsdPrice = Number(apiData.bnt_price.usd);
+    const bntPrice24Change = Number(
+      ((bntUsdPrice / Number(apiData.bnt_price_24h_ago.usd) - 1) * 100).toFixed(
```
Comment (Velua) [inline src/store/modules/swap/ethBancor.ts:1473]: Export this logic to a pureFunction and create a couple tests from Asafs [comment](https://github.com/bancorprotocol/webapp/issues/1050#issue-811127591)

Comment (Ran73Cohen) [reply src/store/modules/swap/ethBancor.ts:1473]: Yeah was planning to its just a draft, waiting on the API to give the rest of the data.

**Comment by ashachaf** [comment]
@Ran73Cohen 
can you update the font based on the mockup?
https://www.figma.com/file/3yED7o7TgoLwoEjV2EkRLL/Bancor-V2?node-id=2674%3A25

**Comment by ashachaf** [comment]
shorter tool tip: "Data refreshes automatically"

**Review by ashachaf — APPROVED** [review]

**Pull request merged**